index.js

```diff
@@ -115,8 +87,10 @@ module.exports = {
   version: version,
   __debug: {
     parse: function(text, opts) {
-      return parse(text, opts);
+      return parser.parse(text, opts);
     },
+    cleanAST: parser.cleanAST,
+    massageAST: parser.massageAST,
```

Comment:
We don't want to expose those two functions in the public npm export. Not only they have a really bad name, but also people shouldn't really do those kind of things.